src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -0,0 +1,247 @@
+
+  * (Recommended) Use the `rook-ceph-tools` Pod to access the ceph CLI. Use the same version of the Rook toolbox as the version of Rook Ceph that is installed in the cluster.
+
+     For information about how to create a `rook-ceph-tools` Pod for Rook Ceph v1.5, see [Rook Toolbox](https://rook.io/docs/rook/v1.5/ceph-toolbox.html) in the Rook Ceph documentation.
+
+  * Use `kubectl exec` to enter the `rook-ceph-operator` Pod, where the ceph CLI is available.
+
+     You can either open an interactive shell to run multiple ceph CLI commands in a row, or you can precede each ceph CLI command that you run with `kubectl exec`. The following shows an example of each method:
+
+ * (Optional) Open an interactive shell in the `rook-ceph-tools` or `rook-ceph-operator` Pod to run multiple ceph CLI commands in a row.
+
+    **Example**:
+
+    Open an interactive shell in the `rook-ceph-tools` or `rook-ceph-operator` Pod:
+
+    ```
+    kubectl exec -it -n rook-ceph rook-ceph-tools-54ff78f9b6-gqsfm -- /bin/bash
+    ```
+    Then, run ceph CLI commands from the interactive shell:
+
+    ```
+    ceph status
+    ```
+
+    If you do not create an interactive shell, precede each ceph CLI command in the `rook-ceph-tools` or `rook-ceph-operator` Pod with `kubectl exec`.
+
+    **Example**:
+
+    ```
+    kubectl exec -it -n rook-ceph rook-ceph-tools-54ff78f9b6-gqsfm -- ceph status
+    ```
+
+* Verify that Ceph is in a healthy state by running one of the following `ceph status` commands in the `rook-ceph-tools` Pod in the `rook-ceph` namespace:
+
+    * **Rook Ceph v1.4.0 or later**:
+
+      ```
+      kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
+      ```
+    * **Rook Ceph v1.0.0 or later**:
+
+      ```
+      kubectl -n rook-ceph exec deploy/rook-ceph-operator -- ceph status
+      ```
+
+      The output of the command shows `health: HEALTH_OK` if Ceph is in a healthy state.
+
+### (Recommended) Manually Rebalance Ceph and Remove a Node
+
+This procedure ensures that the data held in Rook Ceph is safely replicated to a new node before you remove a node. Rebalancing your data is critical for preventing against data loss that can occur when removing a node if the data stored in Ceph has not been properly replicated.
+
+To manually remove a node, you first use the ceph CLI to weigh the Ceph OSD to `0` on the node that you want to remove and wait for Ceph to rebalance the data across OSDs. Then, you can remove the OSD from the node, and finally remove the node.
+
+To manually rebalance data and remove a node:
+
+1. Review the [Rook Ceph and etcd Node Removal Requirements](#rook-ceph-and-etcd-node-removal-requirements) above.
+
+1. Complete the [Rook Ceph Cluster Prerequisites](#rook-ceph-cluster-prerequisites) above.
+
+1. Add the same number of new nodes to the cluster that you intended to remove. For example, if you intend to remove a total of two nodes, add two new nodes.
```

Comment:
```suggestion
1. Add the same number of new nodes to the cluster that you intended to remove.
For example, if you intend to remove a total of two nodes, add two new nodes.
```

Reply:
@jonquil2002 for context, I think this change has to do with Andrew's/the kURL team's preference to have one line of markdown per sentence (for ease of viewing changes in a git diff). Though correct me if I'm wrong about the intent Andrew